uk_bin_collection/uk_bin_collection/councils/SouthOxfordshireCouncil.py

```
@@ -63,53 +63,57 @@ def parse_data(self, page: str, **kwargs) -> dict:
                 # Strip supplementary notes (e.g. "Don't forget...", "Extra garden waste...")
                 # that follow the bin-type description.
                 type_parts = []
                 for part in bin_info[type_start:]:
                     if "don't" in part.lower() or part.startswith("Extra"):
                         break
                     type_parts.append(part)
-                bin_type = str.capitalize(" ".join(type_parts))
-            except:
+                combined_type = " ".join(type_parts)
+            except Exception:
                 continue
```

Ruff (0.16.3)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)